airbyte-cdk/python/airbyte_cdk/sources/file_based/file_types/csv_parser.py

```diff
@@ -132,6 +133,19 @@ class CsvParser(FileTypeParser):
     _MAX_BYTES_PER_FILE_FOR_SCHEMA_INFERENCE = 1_000_000
 
     def __init__(self, csv_reader: Optional[_CsvReader] = None):
+        # Increase the maximum length of data that can be parsed in a single CSV field. The default is 128k, which is typically sufficient
+        # but given the use of Airbyte in loading a large variety of data it is best to allow for the maximum field size possible to avoid
+        # skipping data on load. https://stackoverflow.com/questions/15063936/csv-error-field-larger-than-field-limit-131072
+        max_int = sys.maxsize
```

Comment:
I'm still a little bit uneasy about this block and the cycle. I _think_ this is perfectly fine, but I'm not a Python expert all that much.

But, if you'd agree to just set this to be 2^31, I would honestly be totally fine. I.e. `2,147,483,647 ` is always safe, I think. _Surely 64kb will be enough for everyone._

Reply:
I think that sounds like a good approach. I was also thinking that we can make it a parameter to the function, with the default set to 2^31, leaving the door open to make it configurable in the UI for folks who are abusing their CSVs 🙂